js_modules/dagit/packages/core/src/assets/AssetWipeDialog.tsx

```diff
@@ -51,8 +51,7 @@ export const AssetWipeDialog: React.FC<{
           </ul>
           <div>
             Assets defined only by their historical materializations will disappear from the Asset
-            Catalog. Software-defined assets will remain unless their definition is also deleted
-            from the repository.
```

Comment:
Remove references to repositories if they don't necessarily add usefulness to the copy.